global.json

```diff
@@ -1,5 +1,6 @@
 {
     "sdk": {
-      "version": "5.0.x"
+      "version": "5.0.100",
+      "rollForward": "minor"
```

Comment:
I just tried `minor` on a machine with multiple SDKs installed, and it's using the latest installed 5.0.1xx, instead of 5.0.x as was our original intention:

```
$ dotnet --list-sdks
2.1.302 [/usr/share/dotnet/sdk]
2.1.403 [/usr/share/dotnet/sdk]
2.1.526 [/usr/share/dotnet/sdk]
2.1.617 [/usr/share/dotnet/sdk]
2.1.701 [/usr/share/dotnet/sdk]
2.1.818 [/usr/share/dotnet/sdk]
3.1.119 [/usr/share/dotnet/sdk]
3.1.202 [/usr/share/dotnet/sdk]
3.1.302 [/usr/share/dotnet/sdk]
3.1.413 [/usr/share/dotnet/sdk]
5.0.104 [/usr/share/dotnet/sdk]
5.0.207 [/usr/share/dotnet/sdk]
5.0.303 [/usr/share/dotnet/sdk]
5.0.401 [/usr/share/dotnet/sdk]
$ cat global.json
{
  "sdk": {
    "version": "5.0.100",
    "rollForward": "minor"
  }
}
$ dotnet --version
5.0.104
```

on the same machine with `latestFeature`, it uses the latest installed 5.0.x SDK:
```
$ cat global.json
{
  "sdk": {
    "version": "5.0.100",
    "rollForward": "latestFeature"
  }
}
$ dotnet --version
5.0.401
```

and just to prove the point that it does what we want, here is what it does when pointing to a previous SDK (3.1.x):
```
$ cat global.json
{
  "sdk": {
    "version": "3.1.100",
    "rollForward": "latestFeature"
  }
}
$ dotnet --version
3.1.413
```

and to a previous SDK that is not installed (3.0.x):
```
$ cat global.json
{
  "sdk": {
    "version": "3.0.100",
    "rollForward": "latestFeature"
  }
}
runner@fv-az212-514:~/work/toyrepo/toyrepo$ dotnet --version
Could not execute because the application was not found or a compatible .NET SDK is not installed.
Possible reasons for this include:
  * You intended to execute a .NET program:
      The application '--version' does not exist.
  * You intended to execute a .NET SDK command:
      A compatible installed .NET SDK for global.json version [3.0.100] from [/home/runner/work/toyrepo/toyrepo/global.json] was not found.
      Install the [3.0.100] .NET SDK or update [/home/runner/work/toyrepo/toyrepo/global.json] with an installed .NET SDK:
        2.1.302 [/usr/share/dotnet/sdk]
        2.1.403 [/usr/share/dotnet/sdk]
        2.1.526 [/usr/share/dotnet/sdk]
        2.1.617 [/usr/share/dotnet/sdk]
        2.1.701 [/usr/share/dotnet/sdk]
        2.1.818 [/usr/share/dotnet/sdk]
        3.1.119 [/usr/share/dotnet/sdk]
        3.1.202 [/usr/share/dotnet/sdk]
        3.1.302 [/usr/share/dotnet/sdk]
        3.1.413 [/usr/share/dotnet/sdk]
        5.0.104 [/usr/share/dotnet/sdk]
        5.0.207 [/usr/share/dotnet/sdk]
        5.0.303 [/usr/share/dotnet/sdk]
        5.0.401 [/usr/share/dotnet/sdk]
```

Reply:
Good spot, I got confused by the number of possible options. My intention was to use `latestMinor` (which is broader than the `latestFeature`)